lib/node_modules/@stdlib/math/base/special/besselj0/lib/j0.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+/*
+* The original C++ code and copyright notice is from the [Boost library]{@link https://github.com/boostorg/math/blob/develop/include/boost/math/special_functions/detail/bessel_j0.hpp}.
+*
+* The implementation has been modified for JavaScript.
+*/
+
+/*
+* Copyright John Maddock 2006-7, 2013-14.
```

Comment:
The copyright should be updated for other author, correct?.

Reply:
So it would contain *both* sets of authors? Or just the one included in the particular source file?

```
/*
* Copyright Xiaogang Zhang, 2006.
* Copyright John Maddock 2006-7, 2013-14.
* Copyright Paul A. Bristow 2007, 2013-14.
* Copyright Nikhar Agrawal 2013-14.
* Copyright Christopher Kormanyos 2013-14.
* Use, modification and distribution are subject to the
* Boost Software License, Version 1.0. (See accompanying file
* LICENSE or copy at http://www.boost.org/LICENSE_1_0.txt)
*/
```

vs

```
/*
* Copyright Xiaogang Zhang, 2006.
* Use, modification and distribution are subject to the
* Boost Software License, Version 1.0. (See accompanying file
* LICENSE or copy at http://www.boost.org/LICENSE_1_0.txt)
*/
```

Reply:
I believe just the author in the source file. @Planeshifter ?

Reply:
I've gone with the second option for now.

Reply:
Thanks!

Reply:
Yes, that's what we have done in the past: Just the author of the source file.